docs/topics/auth/customizing.txt

```diff
@@ -547,6 +547,12 @@ password resets. You must then provide some key implementation details:
         value (the :attr:`~django.db.models.Field.primary_key` by default) of an
         existing instance.
 
+    .. attribute:: EMAIL_FIELD
+
+        .. versionadded:: 1.11
+
+        A string describing the name of the email field on the ``User`` model.
```

Comment:
Probably want to add a link to `get_email_field_name()` here once that method is documented.